usageReports/ebs/ebs_route.go

```diff
@@ -0,0 +1,95 @@
+//   Copyright 2018 MSolution.IO
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package ebs
+
+import (
+	"database/sql"
+	"net/http"
+	"time"
+
+	"github.com/trackit/trackit-server/db"
+	"github.com/trackit/trackit-server/routes"
+	"github.com/trackit/trackit-server/users"
+)
+
+type (
+	// EbsQueryParams will store the parsed query params
+	EbsQueryParams struct {
+		AccountList []string
+		IndexList   []string
+		Date        time.Time
+	}
+
+	// EbsUnusedQueryParams will store the parsed query params
+	EbsUnusedQueryParams struct {
+		AccountList []string
+		IndexList   []string
+		Date        time.Time
+		Count       int
+	}
+)
+
+var (
+	// ebsQueryArgs allows to get required queryArgs params
+	ebsQueryArgs = []routes.QueryArg{
+		routes.AwsAccountsOptionalQueryArg,
+		routes.DateQueryArg,
+	}
+
+	// ebsUnusedQueryArgs allows to get required queryArgs params
+	ebsUnusedQueryArgs = []routes.QueryArg{
```

Comment:
This variable is not used, you should remove it.

Reply:
Fixed